templates/README.md

```diff
@@ -0,0 +1,38 @@
+# How to setup AWS Credentials rotation
+
+These files have to be deployed in the platform namespace.
+
+- prerequisite
+
+  ```text
+  - you have access to kubernetes cluster hosting 'platform' namespace.
+  - docker image 'nuxeo/aws-iam-credential-rotate' is available.
+  - create or retrieve credentials from aws:
+    - Access key ID
+    - Secret access key
+  ```
+
+## Create Secret
+
+[secret](secret.yaml) file will be created under the 'platform' namespace:
+
+- aws-iam-user-credentials secret
+
+```bash
+    > $ kubectl create -f templates/secret.yaml
+```
+
+## Create Service Account, Role, Rolebinding and Cronjob
+
+[cronjob](cronjob.yaml) file will create these objects under the 'platform' namespace:
+
+- aws-credentials-updater service account
+- secret-edit role
+- aws-credentials-updater-rolebinding rolebinding
+- rotate-keys cronjob
```

Comment:
Samething about the resource name. Furthermore, it could be interesting to keep the type of created objects and if possible to explain their purposes.

Reply:
I think  this README is just an extract of [AWS IAM key rotate tool](https://github.com/nuxeo-cloud/aws-iam-credential-rotate).

Reply:
I agree  about object name, but for object description it is better to refer to kubernetes docus.